tests/br_clustered_index/run.sh

```diff
@@ -0,0 +1,218 @@
+#!/bin/sh
+#
+# Copyright 2019 PingCAP, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -eu
+DB="$TEST_NAME"
+TABLE="usertable"
+
+run_sql "CREATE DATABASE $DB;"
+
+run_sql "
+USE $DB;
+
+CREATE TABLE t0 (
+	id VARCHAR(255),
+	data INT,
+	PRIMARY KEY(id)
+);
+
+INSERT INTO t0 VALUES ('1', 1);
+INSERT INTO t0 VALUES ('2', 2);
+INSERT INTO t0 VALUES ('3', 3);
+INSERT INTO t0 VALUES ('4', 4);
+INSERT INTO t0 VALUES ('5', 5);
+
+DELETE FROM t0 WHERE id = '3';
+DELETE FROM t0 WHERE data = 5;
+UPDATE t0 SET id = '10' WHERE data = 1;
+UPDATE t0 SET data = 555 WHERE id = '10';
```

Comment:
what's the purpose of doing DELETE + UPDATE followed by INSERT? we are testing BR here, not TiDB.